config/webpack.dev.config.js

```diff
@@ -121,6 +121,7 @@ module.exports = Merge.smart(commonConfig, {
         REPORTING_CONFIGURATIONS: true,
         SUBSCRIPTION_MANAGEMENT: true,
         SUPPORT: true,
+        SAML_CONFIGURATION: true,
```

Comment:
for locally viewing I cannot find the path http://localhost:1991/test-enterprise/admin/samlconfiguration I get a 404 after npm start from this branch, do I need to do any other steps to see this page?

Reply:
Ah sorry, I keep forgetting testing notes. :( Will add them, but you need to check the box in LMS admin for a specific enterprise Customer (that you have a superuser associated with) here: http://localhost:18000/admin/enterprise/enterprisecustomer/ (open a customer > check box near the bottom for "enable portal saml configuration screen" or whatever it says exactly > save > refresh the frontend and you should see it in the sidebar

Reply:
I have not yet figured this out, after creaeting a saml provider config through the admin, I can't find it in the EnterpriseCustomer edit page. I do not want to be a blocker, but I think post this PR I may want to set this up together with you over zoom/slack if you can help me ... sth is off I guess

I may also need to get your edx-platform changes but this looks like a different issue